docs/blog/arcade/game-jam-2.md

```diff
@@ -0,0 +1,57 @@
+
+### NineCarpets by phillowcompiler
+[![NineCarpets](/static/blog/arcade/game-jam/one-room/ninecarpets.png)](https://arcade.makecode.com/81707-47949-81798-72867)
+NineCarpets is a fun pattern-matching game with great graphics, in the style of a tile-sliding puzzle! Use the A button to flip a tile, B to swap two tiles, and pet the cat if you're feeling stressed about time.
+
+### Meteor Defense! by Fieldronix
+[![Meteor Defense!](/static/blog/arcade/game-jam/one-room/meteordefense.png)](https://arcade.makecode.com/77852-25984-35775-89407)
+You are the only line of defense between a deadly meteor shower and your base. Strategically place turrets in Fieldronix's Meteor Defense! to survive for thirty seconds.
+
+### Guerrein by Alejandro
+[![Guerrein](/static/blog/arcade/game-jam/one-room/guerrein.png)](https://arcade.makecode.com/29746-46301-89977-97050)
+Protect the earth! Play as a space warrior defending the earth from invading robots in Alejandro's Guerrein. Survive for as long as you can, and collect coins while protecting the planet.
+
+### Aqua Ball by Perryw101
+[![Aqua Ball](/static/blog/arcade/game-jam/one-room/aquaball.png)](https://arcade.makecode.com/00907-50718-83595-32559)
+It's the annual game of Aqua Ball in Fishville, and you're up against the reigning champ. Perryw101's Aqua Ball stars Goldson, an underdog, trying to take down The Shark, an Aqua Ball champion who may have some dirty tricks up their sleeve.
+
+### Wave Jumper by shaqattack8
+[![Wave Jumper](/static/blog/arcade/game-jam/one-room/wavejumper.png)](https://arcade.makecode.com/15157-22582-17393-73487)
+Totally tubular! Rack up combo points by riding the waves as long as possible in shaqattack8's Wave Jumper. Grab a friend and try out multiplayer mode for an extra challenge.
+
+### Clicking Button Simulator by o_danny
+[![Clicking Button Simulator](/static/blog/arcade/game-jam/one-room/clickingbutton.png)](https://arcade.makecode.com/98143-93052-87735-45035)
+What!? I have to click this button *one thousand* times?
+Fear not, o_danny's game has more going on than you might think.
+We don't want to spoil anything, so give it a shot!
+
+### Gigantic Castle by reyhanpanci256
+[![Gigantic Castle](/static/blog/arcade/game-jam/one-room/giganticcastle.png)](https://arcade.makecode.com/99246-16626-78826-85418)
+Gigantic Castle is an action platformer with multiple ways to play.
+Choose between two different paths as you navigate a castle stocked with terrifying ghosts, monsters, spikes, and potted plants.
+This game is a great example of level-design done well!
+reyhanpanci is a frequent contributor on the [MakeCode Forum](forum.makecode.com), make sure you check out their other games!
```

Comment:
```suggestion
reyhanpanci is a frequent contributor on the [MakeCode Forum](https://forum.makecode.com), make sure you check out their [other games](https://forum.makecode.com/search?q=user%3Areyhanpanci256%20tags%3Agame)!
```

maybe on the link to the search for their other games, but I think as is the makecode forum link will be relative (=> `makecode.com/forum.makecode.com`) so that needs https